packages/console/src/scss/normalized.scss

```diff
@@ -33,24 +33,14 @@ table {
       padding: _.unit(3);
       border-bottom: 1px solid var(--color-divider);
       text-align: left;
-
-      &:first-child,
-      &:last-child {
-        padding: _.unit(3) _.unit(8);
-      }
     }
   }
 
   tbody {
     td {
       font: var(--font-body-medium);
       border-bottom: 1px solid var(--color-divider);
-      padding: _.unit(3) _.unit(2);
-
-      &:first-child,
-      &:last-child {
-        padding: _.unit(3) _.unit(8);
-      }
+      padding: _.unit(3);
```

Comment:
Put all table related css styles in the new Table component stylesheet

Reply:
Will do it in later PRs, for we don't replace all tables in this PR.

Reply:
Ah, I see. OK then, as long as it is planned